apps/mudita-center-e2e/src/page-objects/contacts-kompakt.ts

```diff
@@ -30,31 +30,75 @@ class ContactsKompaktPage extends Page {
   }
 
   public get contactsTableRow() {
-    return $('//*[@data-testid="ui-table-row"]')
+    return $('//*[@data-testid="ui-table-cell"]')
```

Comment:
Do you think this change is really necessary? If so, doesn’t it require renaming the method?